src/modules/test/items-count.js

```diff
@@ -0,0 +1,7 @@
+const mealCounter = document.getElementById('meals');
+
+export default function countMeals(meals) {
+  if (mealCounter !== null) {
+    mealCounter.innerHTML = `<p>Meals(${meals.length})</p>`;
+  }
+}
```

Comment:
- [ ] As suggested by previous reviewer please use arrow functions instead of function keyword as its been depreciated.